examples/webapp/webapp.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+import asyncio
+from datetime import timedelta
+import pathlib
+import sys
+
+
+from datetime import datetime
+
+from yapapi import (
+    Golem,
+)
+from yapapi.log import enable_default_logger, log_summary, log_event_repr  # noqa
+from yapapi.payload import vm
+from yapapi.services import Service, ServiceState
+
+examples_dir = pathlib.Path(__file__).resolve().parent.parent
+sys.path.append(str(examples_dir))
+
+from utils import (
+    build_parser,
+    TEXT_COLOR_CYAN,
+    TEXT_COLOR_DEFAULT,
+    run_golem_example,
+    print_env_info,
+)
+from utils.service.http_proxy import HttpProxyService, LocalHttpProxy
+
+HTTP_IMAGE_HASH = "c37c1364f637c199fe710ca62241ff486db92c875b786814c6030aa1"
+DB_IMAGE_HASH = "85021afecf51687ecae8bdc21e10f3b11b82d2e3b169ba44e177340c"
+
+STARTING_TIMEOUT = timedelta(minutes=4)
+
+
+class HttpService(HttpProxyService):
+    def __init__(self, db_address: str, db_port: int = 4001):
+        super().__init__(remote_port=5000)
+        self._db_address = db_address
+        self._db_port = db_port
+
+    @staticmethod
+    async def get_payload():
+        return await vm.repo(
+            image_hash=HTTP_IMAGE_HASH,
+            capabilities=[vm.VM_CAPS_VPN],
+        )
+
+    async def start(self):
+        # perform the initialization of the Service
+        # (which includes sending the network details within the `deploy` command)
+        async for script in super().start():
+            yield script
+
+        script = self._ctx.new_script(timeout=timedelta(seconds=10))
+
+        script.run(
+            "/bin/bash",
+            "-c",
+            f"cd /webapp && python app.py --db-address {self._db_address} --db-port {self._db_port} initdb",
+        )
+        script.run(
+            "/bin/bash",
+            "-c",
+            f"cd /webapp && python app.py --db-address {self._db_address} --db-port {self._db_port} run > /webapp/out 2> /webapp/err &",
+        )
```

Comment:
As we already have `run_rqlite.sh`, why not also `run_webapp.sh`? Will be more readable (and also now one might wonder why this is done in two different ways).

Reply:
well, I don't feel there needs to be a similarity here ... those are two different images... and here, the script has two operations that can be done -> `initdb` and `run` ... while in this example we do want both to happen, it's not a given, default operation for this image... so I'd leave it be ...

Reply:
Well, I can't say I fully agree, but I see your point. I'd still prefer a separate `.sh`, but I don't insist.